tests/build/depfile_cython.srctree

```diff
@@ -0,0 +1,61 @@
+"""
+PYTHON -c 'import os; os.makedirs("builddir/pkg/sub")'
+CYTHON -M pkg/test.pyx -o builddir/pkg/test.c
+CYTHON -M pkg/sub/test.pyx -o builddir/pkg/sub/test.c
```

Comment:
Why not test both while we're at it?
```suggestion
CYTHON -M pkg/test.pyx -o builddir/pkg/test.c
CYTHON --depfile pkg/sub/test.pyx -o builddir/pkg/sub/test.c
```

Reply:
Well, it's basically an argparse test, but fair enough, it cannot possibly hurt and is literally free.